extension-manifest-v3/src/pages/settings/components/devtools.js

```diff
@@ -46,8 +46,11 @@ function clearStorage(host, event) {
       chrome.storage.local.clear();
 
       // Remove all indexedDBs
-      const dbs = await indexedDB.databases();
-      await Promise.all(dbs.map((db) => indexedDB.deleteDatabase(db.name)));
+      // Note: Firefox doesn't support indexedDB.databases()
```

Comment:
for Firefox we should still delete known databases

Reply:
I'll look at this tomorrow

Reply:
@chrmod @philipp-classen This change is not related to the main PR goal - I have noticed an error message when testing in Firefox. The arguments are correct, but I made a separate issue for that - https://github.com/ghostery/ghostery-extension/issues/1090